canonicalwebteam/discourse_docs/__init__.py

```diff
@@ -1,3 +1,9 @@
-from canonicalwebteam.discourse_docs.app import DiscourseDocs  # noqa
+from canonicalwebteam.discourse_docs.app import (  # noqa
+    Discourse,  # noqa
```

Comment:
I assume the `Discourse` object is only for docs right? Since `EngagePages` is for engage pages. So can we call this object `Docs` please?